OsmAnd/res/values/strings.xml

```diff
@@ -11,21 +11,21 @@
 	Thx - Hardy
 
 -->
-    <string name="permission_is_required">Permission is required to use this option.</string>
-    <string name="logcat_buffer_descr">Check and share detailed logs of the application</string>
+    <string name="permission_is_required">You must grant the required permission to use this.</string>
```

Comment:
Not better, but longer than the original?

Reply:
The convention is to instruct directly. It is easier to understand because it has less abstraction. I would go "to make use of", but that is more British-English.  "Permission is required" is un/referenced noun uncanny wally, should have been "permission required" or "A permission is required". If having to add "to use this option", might as-well lower all barriers and directly instruct the user in what to do.

Reply:
Yes, being specific would be best here. Have not checked the context and what is known there. How about 'Please garnt the required permission to use this'? But then, have you checked when this occurs and how the user is even notified which permission it is?